<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLDotMatrixDisplayWizardBase</class>
 <widget class="QDialog" name="KLDotMatrixDisplayWizardBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dot matrix display wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QCheckBox" name="cbAddLibrary">
     <property name="text">
      <string>Add HD&amp;44780 library by Peter Fleury to the project</string>
     </property>
     <property name="shortcut">
      <string>Alt+4</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>11</width>
         <height>280</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="tlValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>XTAL:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>F_CPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLSpinBoxWidget" name="sbValue"/>
         </item>
         <item>
          <widget class="QComboBox" name="cbValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="spacer12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>196</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="urlHomepage" native="true">
         <property name="text" stdset="0">
          <string>See http://www.jump.to/fleury for details.</string>
         </property>
         <property name="url" stdset="0">
          <string>http://homepage.hispeed.ch/peterfleury/group__pfleury__lcd.html</string>
         </property>
         <widget class="KUrlLabel" name="lvValues" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>521</width>
            <height>80</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>362</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Ca&amp;ncel</string>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KLSpinBoxWidget</class>
   <extends>QSpinBox</extends>
   <header location="global">klspinboxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QWidget</extends>
   <header>kurllabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbAddLibrary</tabstop>
  <tabstop>urlHomepage</tabstop>
  <tabstop>leValue</tabstop>
  <tabstop>cbValue</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <includes>
  <include location="local">kurllabel.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>KLDotMatrixDisplayWizardBase</receiver>
   <slot>slotOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>KLDotMatrixDisplayWizardBase</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAddLibrary</sender>
   <signal>toggled(bool)</signal>
   <receiver>KLDotMatrixDisplayWizardBase</receiver>
   <slot>slotAddHD44780Lib(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KLDotMatrixDisplayWizardBase</receiver>
   <slot>slotChangeValue(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbValue</sender>
   <signal>activated(QString)</signal>
   <receiver>KLDotMatrixDisplayWizardBase</receiver>
   <slot>slotChangeValue(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
